packages/git/src/browser/diff/git-diff-contribution.ts

```diff
@@ -101,6 +111,33 @@ export class GitDiffContribution extends AbstractViewContribution<GitDiffWidget>
         }));
     }
 
+    registerToolbarItems(registry: TabBarToolbarRegistry): void {
+        // Register 'more actions...' toolbar item which consists of multiple commands.
+        const navigatorRegisterItem = (item: Mutable<TabBarToolbarItem>) => {
```

Comment:
Can we add a method on `FileNavigatorContribution`, inject it here, and call instead of copy pasting code?

```ts
@inject(FileNavigatorContribution)
protected readonly navigatorContribution: FileNavigatorContribution;

registerToolbarItems(): void {
    this.navigatorContribution.registerToolbarItem({
        //....
    });
}